src/index.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable comma-dangle */
```

Comment:
- [ ] `required`
- [ ] Customizing the Linters by `Disabling the eslint` defeats the purpose of using it please enable it.

Reply:
@mwanawabangona here is the same except I tried to fix the comma dangle and I couldn't . But also this is in the index.js this comes with react I didn't write that code

Reply:
@david63011 For this kindly you need to fix the comma dangle. Kindly just because you didn't write the code doesn't exempt you from fixing it. Remember, linters are there to make sure your code is clean.